oeps/processes/oep-0057-core-repositories.rst

```diff
@@ -0,0 +1,166 @@
+
+* defines an administrative boundary within which the Core Contributor Program applies;
+
+* makes the Open edX project more comprehensible by collecting its most critical components;
+
+* (TODO: should I add something about tCRIL stewardship/ownership?); and
+
+* encompasses the first-party aspect of the Open edX release supply chain, identifying repositories that need to be:
+
+  * monitoried for discloures under the `Open edX security policy`_,
+  * regularly updated with security patches,
+  * covered by automated unit tests and static analysis, and
+  * ready to deploy to production at any time.
+
+
+Considerations
+--------------
+
+
+* Resolving legacy processes and access policies
+
+  * The process for reviewing addition/removal of repos via the Core Contributor program isn't ready to be spun up yet or even well-defined. In the meantime, tCRIL has been making these judgement calls.
+  * 2U OCM and some of tCRIL retain historical access to many Core repositories outside of the Core Contributor program process. There is a plan to bring those folks under the Core Contributor umbrella, but it will take time.
+  * tCRIL is currently using a few public repositories in the openedx organization for day-to-day work, such as ``openedx/tcril-engineering``. Upon acceptance of this OEP, they would need to make a new GitHub organization for those repositories.
+
+* Cleaning up the Core
+
+  * Many repositories in the openedx GitHub organization do not currently meet the Core criteria. We will need to review the organization contents, and for several repos, transfer them out or invest in them.
+  * Even so, the definition of the Core still captures a large number of repositories, probably larger than we want our "Core" to be. We could reduce it by loosening the package dependencies of many services to being optional instead of strictly required.
+
+* Ensuring the Core contains everything it should
+
+  * There are four repositories directly included in Maple that aren't in the openedx organization. We'll either need to remove them from the release or transfer them into openedx.
+  * This will take a while, but we'll need to look at the list of packages that are depended upon by released repositories. If any of them are Open edX code but outside the openedx organization, we'll need to move them into openedx, or refactor the dependencies so that they're optional.
+
+
+
+Motivation
+----------
+
+TODO
+
+
+Alternatives Considered
+-----------------------
+
+TODO
+
+
+Change History
+--------------
+
+2022-03-XX
+==========
+
+* Document created in `initial pull request`_.
+
+
+.. _openedx GitHub organization: https://github.com/openedx
+.. _security policy: https://github.com/openedx/edx-platform/security/policy
+.. _latest Open edX release: ./oep-0010-proc-openedx-releases.rst
+.. _maintainers: ./oep-0055-project-maintainers.rst
+.. _Core Contributor Program: ./oep-0054-core-contributors.rst
```

Comment:
Q: does this relative linking work in the RTD site? I've only seen explicit links in other OEPs.